tmt/base.py

```diff
@@ -3113,12 +3114,12 @@ def init(
                     path = tree.root
                 except fmf.utils.GeneralError as error:
                     raise tmt.utils.GeneralError(
-                        f"Failed to initialize tree in '{path}': {error}")
-                echo(f"Tree '{tree.root}' initialized.")
+                        f"Failed to create '{fmf_dir}': {error}")
+                echo(f"Initialized metadata dir '{fmf_dir}'")
+                echo(f"Add tests with `create test` or `init --template`")
```

Comment:
I for one would include `tmt` in those two hints, `tmt create test`. Itfeels more complete, less ambiguity (“is it `init` or am I supposed to add tmt implicitly?”)

Reply:
Added `tmt`. There was an error, though, - the real command is `tmt test create`. Fixed.